MagisterkaApp.UI/ViewModel/FrequenceStepsViewModel.cs

```diff
@@ -118,9 +114,12 @@ public void Check5proc()
                    if (saveWindow.ShowDialog() == System.Windows.Forms.DialogResult.OK)
                    {
                        string savePath = Path.GetDirectoryName(saveWindow.FileName);
+                       var savePathTest = savePath + $"\\{measure.NameOfMeasure}Test.txt";
                        savePath = savePath + $"\\{measure.NameOfMeasure}.DAT";
+                    
 
                        SaveResult.WriteResult(filtredFrequencySteps, savePath, measure);
+                       SaveResult.WriteTestValues(new List<FrequencyStep>(this.FrequencySteps), savePathTest);
```

Comment:
This shouldn't be active. We don't need to have .txt file after each save. Comment it.